src/resources/README.md

```diff
@@ -1,2 +1,11 @@
 Always use `getResourcePath()` to get the appropriate path to these resources depending
 on the context (WPT, standalone, worker, etc.)
+
+
+The test video files were generated from https://source.chromium.org/chromium/chromium/src/+/main:media/test/data/four-colors.png with the ffmpeg cmds below:
```

Comment:
nit: Consider just including the four-colors.png file. I don't think we should hard link chromium if we don't have to.

Reply:
Done!